components/brave_ads/core/internal/creatives/inline_content_ads/creative_inline_content_ads_database_table.cc

```diff
@@ -487,11 +477,11 @@ void CreativeInlineContentAds::Create(mojom::DBTransactionInfo* transaction) {
   mojom::DBCommandInfoPtr command = mojom::DBCommandInfo::New();
   command->type = mojom::DBCommandInfo::Type::EXECUTE;
   command->sql =
-      "CREATE TABLE creative_inline_content_ads "
```

Comment:
I think it would be good as a follow-up PR to re-format all our SQL queries to make them more readable.

Reply:
Totally agree, raised https://github.com/brave/brave-browser/issues/32742. Thanks